include/ck_tile/ops/grouped_convolution/kernel/grouped_convolution_backward_data_kernel.hpp

```diff
@@ -779,20 +796,20 @@ struct GroupedConvolutionBackwardDataKernel
     }
 
     template <typename TensorView>
-    CK_TILE_DEVICE static auto MakeGemmPadViews(const TensorView& views)
+    CK_TILE_DEVICE static auto MakeGemmPadViews(const TensorView& views, const index_t k_batch)
     {
         const auto& a_pad_view = [&]() {
             const auto& a_tensor_view = views.at(I0);
             return pad_tensor_view(a_tensor_view,
                                    make_tuple(number<TilePartitioner::MPerBlock>{},
-                                              number<TilePartitioner::KPerBlock>{}),
+                                              number<TilePartitioner::KPerBlock>{} * k_batch),
                                    sequence<true, true>{});
         }();
 
         const auto& b_pad_view = [&]() {
             const auto& b_tensor_view = views.at(I1);
             return pad_tensor_view(b_tensor_view,
-                                   make_tuple(number<TilePartitioner::KPerBlock>{},
+                                   make_tuple(number<TilePartitioner::KPerBlock>{} * k_batch,
```

Comment:
k_batch is number of K division. So it is not multiplayer for padding it should be div_up(K / KPerBlock, k_batch) * KPerBlock . I see we have the some issue in backward_weight_kernel. Could you fix this?